Sources/Core/Selection/Views/SelectionTagsContainerView.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+protocol SelectionTagsContainerViewDelegate: AnyObject {
+    func selectionTagsContainerView(_ view: SelectionTagsContainerView, didRemoveTagAt index: Int)
+}
+
+final class SelectionTagsContainerView: UIView {
+    weak var delegate: SelectionTagsContainerViewDelegate?
+
+    // MARK: - Private properties
+
+    private lazy var collapsedView = SelectionTagView(withAutoLayout: true)
+
+    private lazy var expandedView: UIStackView = {
```

Comment:
I think we could have a better name for this. It is not really expanded, it is just not collapsed :P maybe something like `selectionTagStackView` instead, or simply `stackView`. I feel like this makes it more clear what the `collapsedView` really is.

Reply:
Makes sense. `stackView` it is 👍